producers/http/handlers.go

```diff
@@ -27,19 +27,15 @@ import (
 
 func nodeHandler(p *producerImpl) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		var am []interface{}
 		nodeMetrics, err := p.store.GetByRegex(producers.NodeMetricPrefix + ".*")
```

Comment:
I know this wasn't part of your changeset, but we should really be escaping any constant strings here that go into `store.GetByRegex` here and elsewhere.